openshift-gitops/infra/base/projects.yaml

```diff
@@ -3,30 +3,30 @@ apiVersion: argoproj.io/v1alpha1
 kind: AppProject
 metadata:
   name: dataplane
-  namespace: openshift-gitops
```

Comment:
Why are you deleting the `namespace: openshift-gitops`

Reply:
because it is part of the kustomization file and will be added to all the resources

Reply:
But we are not kustomizing it as such though it should work either ways.

Reply:
we're now using kustomize to deploy the projects